src/views/credits/people.json

```diff
@@ -234,4 +234,9 @@
         "userId": 26779669,
         "name": "Kathy"
     }
```

Comment:
There needs to be a comma here after the closing curly bracket.

Reply:
I agree.